smartsim/_core/mli/comm/channel/dragon_fli.py

```diff
@@ -93,6 +98,8 @@ def recv(self, timeout: float = 0.001) -> t.List[bytes]:
                     )
                 except fli.FLIEOT:
                     eot = True
+                except Exception:
+                    break
```

Comment:
Should we re-raise?

Reply:
I was going back and forth on this, but I think you're right. I was thinking if there was an error that occurred after we got some messages, then we should just send the messages that we have. But really we should be informing the user that there was an error because the data will be incomplete and possibly break down the road. Fixing!